src/lib/util.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var $ = require('$jquery');
```

Comment:
Agreed. This version of HUD is pretty much copy and paste from HUD in Glimpse v1. I would hope that we can remove this dependency moving forward, but isn't a priority for this sprint. Here we are simply trying to update the style.